cdap-distributions/src/hdinsight/mainTemplate.json

```diff
@@ -50,18 +50,18 @@
       },
       "installScriptActions": [{
         "name": "[concat('cdap-pageblob-configure-v0','-' ,uniquestring(variables('applicationName')))]",
-        "uri": "https://raw.githubusercontent.com/caskdata/cdap/release/3.5/cdap-distributions/src/hdinsight/pageblob-configure.sh",
+        "uri": "http://downloads.cask.co/hdinsight/pageblob-configure-3.5.1.sh",
```

Comment:
should this be updated to `http://downloads.cask.co/hdinsight/pageblob-configure-3.6.0.sh`? When is this script created?